src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure/Commands/RunCommand/BaseSuite/ReliabilityFramework/TestingScript.sh.txt

```diff
@@ -0,0 +1,46 @@
+# create output folder
```

Comment:
Does this and the ps1 equivalent have to be a .txt file? Can it be a .sh and .ps1 respectively